cardano-cli/test/cardano-cli-golden/files/golden/key/non-extended-keys/non-extended-drep.vkey

```diff
@@ -0,0 +1,5 @@
+{
+    "type": "DRepVerificationKey_ed25519",
+    "description": "",
```

Comment:
Should we have a description for that key here?

Reply:
@carbolymer> I added a description in commit `drep verification key: extended->non extended: specify description`. Is that what you had in mind?

I did the minimal change to avoid disturbing existing tests and code.

Reply:
Sorry, I should've been more explicit. It's an open question from my side whether we should have a description here.
@CarlosLopezDeLara what do you think, should we put a description here, and which one?

Reply:
"Delegate Representative Verification Key"

Reply:
I made the constant a top-level symbol of `Cardano.CLI.EraBased.Run.Key` and reused it from `Governance`. I looked around but saw no obvious destinations and didn't want to introduce a `Constant` module.

@carbolymer> let me know if that suits you.

I also propose to put descriptions when converting other types of extended keys, but will do that in another PR, to avoid delaying this one.